bluemira/radiation_transport/radiation_profile.py

```diff
@@ -410,64 +422,141 @@ def plot_mp_radiation_profile(self):
         """
         self.mp_profile_plot(self.profiles.psi_n, self.rad_mp, self.impurity_symbols)
 
-    def calculate_core_distribution(self) -> list[list[np.ndarray]]:
+    def calculate_core_distribution(self, scaling) -> list[list[np.ndarray]]:
         """
         Build poloidal distribution (distribution along the field lines) of
         line radiation loss in the plasma core.
 
         Returns
         -------
-        rad:
-            Line core radiation.
-            For specie and each closed flux line in the core
+        rad :
+            Line core radiation for each impurity species
+            and for each closed flux line in the core.
         """
-        # Closed flux tubes within the separatrix
+        # Collect closed flux tubes within the separatrix
         self.flux_tubes = self.collect_flux_tubes(self.profiles.psi_n)
 
-        # For each flux tube, poloidal density profile.
-        self.ne_pol = [
-            self.flux_tube_pol_n(ft, n, core=True)
-            for ft, n in zip(self.flux_tubes, self.profiles.ne, strict=False)
-        ]
+        if scaling:
+            # Calculate scaling factors for temperature and density along each flux tube
+            self.scaling_pol_t = [
+                self.scaling_factor_core(ft, alpha=0.5) for ft in self.flux_tubes
+            ]
+            self.scaling_pol_n = [
+                self.scaling_factor_core(ft, alpha=0.5) for ft in self.flux_tubes
+            ]
```

Comment:
these lines are the same unless I'm missing something? Could we just copy?

Reply:
@je-cook I wrote two lines because alpha may be different between temperature and density. Should I set it in a different way?

Reply:
but here alpha is hardcoded to it will never be different? 
The lines themselves are identical apart from the variable name

Reply:
I am thinking of removing these scaling factors. They make very minor difference in the overall calculation, and hardcoded are even more useless.